compliance_checker/cf/util.py

```diff
@@ -261,7 +262,7 @@ def _get(self, entrynode, attrname, required=False):
     def __init__(self, cached_location=None):
         if cached_location:
             try:
-                with open(cached_location, 'r') as fp:
+                with io.open(cached_location, 'r', encoding='utf-8') as fp:
```

Comment:
I don't really want to get into the "debate" :wink: 

My concern is: if the goal is to allow the user to use a custom downloaded table then we should not have a fallback or the user will never know which file is being used.